README.md

```diff
@@ -90,6 +90,27 @@ The following environment variables need to be set in the `.env` file
 - `INSIGHTLY_HOST`
 - `INSIGHTLY_API_KEY`
 
+## Horizon
+
+### Configuration
+
+By default Laravel processes jobs synchronously, because `QUEUE_CONNECTION` is set to `sync`.
+
+To enable Horizon, set the `QUEUE_CONNECTION` environment variable to `redis` in the `.env` file.
+
+Once asynchronous processing is enabled, you also need to start the queues with `make horizon`.
+
+### Monitoring
+
+Horizon is a dashboard for monitoring the queue. It can be accessed at [http://localhost/horizon](http://localhost/horizon).
```

Comment:
```suggestion
Horizon has a dashboard for monitoring the queue. It can be accessed at [http://localhost/horizon](http://localhost/horizon).
```

Unless it is "is" of course